src/hooks/UrqlUseSubscription.re

```diff
@@ -18,20 +20,13 @@ type useSubscriptionResponseJs('ret) = {
   error: UrqlCombinedError.t,
 };
 
-type useSubscriptionResponse('ret) = {
-  fetching: bool,
-  data: option('ret),
-  error: option(UrqlCombinedError.t),
-  response: UrqlTypes.response('ret),
-};
-
 [@bs.module "urql"]
 external useSubscriptionJs:
   (useSubscriptionArgs, option((option('acc), Js.Json.t) => 'acc)) =>
   array(useSubscriptionResponseJs('ret)) =
   "useSubscription";
 
-let useSubscriptionResponseToRecord = (parse, result) => {
+let useSubscriptionResponseToRecord = (parse, result): hookResponse('a) => {
```

Comment:
+1 to using `'ret` in lieu of `'a` here to make it easier to track type parameters.